charts/mev-boost/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for mev-boost.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Options: goerli-shadow-fork-5, ropsten, kiln, sepolia
+network: "ropsten"
+
+# Relay URLs
+relays:
+  goerli-shadow-fork-5: "https://0xafa4c6985aa049fb79dd37010438cfebeb0f2bd42b115b89dd678dab0670c1de38da0c4e9138c9290a398ecd9a0b3110@builder-relay-goerli-sf5.flashbots.net"
+  ropsten: "https://0xb124d80a00b80815397b4e7f1f05377ccc83aeeceb6be87963ba3649f1e6efa32ca870a88845917ec3f26a8e2aa25c77@builder-relay-ropsten.flashbots.net"
+  kiln: "https://0xb5246e299aeb782fbc7c91b41b3284245b1ed5206134b0028b81dfb974e5900616c67847c2354479934fc4bb75519ee1@builder-relay-kiln.flashbots.net"
+  sepolia: "https://0x845bd072b7cd566f02faeb0a4033ce9399e42839ced64e8b2adcfc859ed1e8e1a5a293336a49feac6d9a5edb779be53a@builder-relay-sepolia.flashbots.net"
```

Comment:
Not sure whether we can do something here, but there could be a case when the relay stops responding. Is there a way we could failover to the internal block builder in such a case?

Reply:
@tsudmi if mev-boost is unavailable client will use internal block builder